Makefile

```diff
@@ -54,6 +54,10 @@ LD_FLAGS += -X k8s.io/component-base/version.gitMajor=$(KUBECTL_MAJOR)
 LD_FLAGS += -X k8s.io/component-base/version.gitMinor=$(KUBECTL_MINOR)
 LD_FLAGS += -X k8s.io/component-base/version.buildDate=$(BUILD_DATE)
 LD_FLAGS += -X k8s.io/component-base/version.gitCommit=not_available
+LD_FLAGS += -X github.com/containerd/containerd/version.Version=v$(containerd_version)
+ifeq ($(EMBEDDED_BINS_BUILDMODE), docker)
+LD_FLAGS += -X github.com/containerd/containerd/version.Revision=$(shell ./embedded-bins/staging/linux/bin/containerd --version | awk '{print $$4}')
```

Comment:
Doesn't work for the Windows build. (Don't know if we care that much about it right now.)

> /bin/sh: 1: ./embedded-bins/staging/linux/bin/containerd: not found

Reply:
`k0s ctr ...` is not usable at all in Windows so I think this does not matter. On Windows we currently support only docker as the runtime with which `k0s ctr ...` is not really usable at all anyways.